rhel7/profiles/C2S.profile

```diff
@@ -78,6 +78,7 @@ selections:
     - service_xinetd_disabled
     - service_chronyd_or_ntpd_enabled
     - package_xorg-x11-server-common_removed
+    - xwindows_runlevel_target
```

Comment:
Change reverted in https://github.com/ComplianceAsCode/content/commit/ab0a9490dcab448372b93e1364b5b4570e4d6630